compact_str/src/lib.rs

```diff
@@ -16,20 +16,15 @@
 use core::borrow::Borrow;
 use core::cmp::Ordering;
 use core::fmt;
-use core::hash::{
-    Hash,
-    Hasher,
-};
+use core::hash::{Hash, Hasher};
```

Comment:
Can you re-run `cargo +nightly fmt`? We use some rules which are nightly only

Reply:
That explains why the CI complains about the code style, while my neovim is configured to automatically format my code using `cargo fmt`.

I will run `cargo +nightly fmt` just before merging so that I don't have to run it repeatively.

Reply:
I've applied `cargo +nightly fmt --all`.

SInce this PR is pretty close to being merged, I would apply `cargo +nightly fmt --all` everytime I push new commits.